library/src/main/java/com/goddb/internal/DBImpl.java

```diff
@@ -159,6 +159,47 @@ public void del(String path, String condition) throws GoddbException {
             }
         }
     }
+    // ***********************
+    // *      UPDATE
+    // ***********************
+
+    @Override
+    public void update(String path, String condition, String modData) throws GoddbException {
+        boolean chk = false;
+        String[] section = modData.split(",");
+
+        ArrayList<Integer> wildcardArrayList = Wildcard.extractWildcard(path, mappingTable);
+
+        for (int curPath : wildcardArrayList) {
+            try {
+                JSONArray oldArray = new JSONArray(__get(String.valueOf(curPath)));
+                JSONArray conditionArray, retArray = new JSONArray();
```

Comment:
conditionArray doesn't need to create an object.
and should remove to retArray.